catalog/MDCCatalog.xcodeproj/xcshareddata/xcschemes/MDCCatalog.xcscheme

```diff
@@ -587,12 +587,22 @@
             skipped = "NO">
             <BuildableReference
                BuildableIdentifier = "primary"
-               BlueprintIdentifier = "45A0F5F05D024D63A454A981932552B0"
+               BlueprintIdentifier = "032F1E326A2F5B54A52350BE13F3EC5C"
                BuildableName = "MaterialComponentsSnapshotTests-Unit-SnapshotTests.xctest"
                BlueprintName = "MaterialComponentsSnapshotTests-Unit-SnapshotTests"
                ReferencedContainer = "container:Pods/Pods.xcodeproj">
             </BuildableReference>
          </TestableReference>
+         <TestableReference
+            skipped = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "F413A0F53D9A356F61C27680636F29DC"
+               BuildableName = "MaterialComponentsBeta-Unit-TextFields+Theming-UnitTests.xctest"
+               BlueprintName = "MaterialComponentsBeta-Unit-TextFields+Theming-UnitTests"
```

Comment:
Shouldn't this an AppBar both be in Dragons?

Reply:
Filed https://github.com/material-components/material-components-ios/issues/6935 to investigate and correct this.

Reply:
https://github.com/material-components/material-components-ios/pull/6936 has the fix.